doc/sdk-cli-reference.md

```diff
@@ -296,7 +296,7 @@ should use `up local` instead.
 
```

Comment:
Out of this PR but was thinking should we at some point generate this documentation instead and put it in CI to make sure it's up to date?

Reply:
We could probably do that. There's a couple of formatting stuff that I tend to change in the CLI reference vs the CLI command's help itself, but it shouldn't be too difficult IMO. And now that you mention it, the CLI reference is missing new flags for some subcommands (like `test local`), so it would definitely be good to implement.